js-core-2/lesson7.md

```diff
@@ -32,59 +32,60 @@ When we are you solving a problem, one of your main decisions as a developer is
 
```

Comment:
`hasOwnProperty` is tough to cover without going into more detail around prototypes (which I understand are later in the lesson? or have they covered them already to an extent?). I've added an exercise using it but i'm not sure if it's really adding much value...

I'll have a think about the dynamic property stuff tonight - I'm hesitant to set anything in stone though. I'd rather try to adapt the lesson as they go.